meerk40t/gui/wxmribbon.py

```diff
@@ -665,11 +671,11 @@ def _create_signal_toggler(self, button, signal):
         def make_toggle_click(_tb):
             def toggle_click(origin, set_value, *args):
                 if set_value:
-                    _tb.toggle = False
-                    self._restore_button_aspect(_tb, _tb.state_unpressed)
-                else:
                     _tb.toggle = True
                     self._restore_button_aspect(_tb, _tb.state_pressed)
+                else:
+                    _tb.toggle = False
+                    self._restore_button_aspect(_tb, _tb.state_unpressed)
```

Comment:
The sent state, `set_value` is specifically applied to the toggle to set the same state as the given value.